editor/src/clj/editor/gl/texture.clj

```diff
@@ -28,34 +30,37 @@
            [java.nio Buffer ByteBuffer]))
 
 (set! *warn-on-reflection* true)
+(set! *unchecked-math* :warn-on-boxed)
 
 (def
   ^{:doc "Special constant used as the page-count for non-paged textures.
           A texture can be paged, but only have one page. A value of zero means
           it is not a paged texture. Built as TYPE_2D, not TYPE_2D_ARRAY."}
   ^:const ^:long non-paged-page-count 0)
 
-(def
-  ^{:doc "This map translates Clojure keywords into OpenGL constants.
-          You can use the keywords in texture parameter maps."}
-  texture-params
-  {:base-level   GL2/GL_TEXTURE_BASE_LEVEL
-   :border-color GL2/GL_TEXTURE_BORDER_COLOR
-   :compare-func GL2/GL_TEXTURE_COMPARE_FUNC
-   :compare-mode GL2/GL_TEXTURE_COMPARE_MODE
-   :lod-bias     GL2/GL_TEXTURE_LOD_BIAS
-   :min-filter   GL/GL_TEXTURE_MIN_FILTER
-   :mag-filter   GL/GL_TEXTURE_MAG_FILTER
-   :min-lod      GL2/GL_TEXTURE_MIN_LOD
-   :max-lod      GL2/GL_TEXTURE_MAX_LOD
-   :max-level    GL2/GL_TEXTURE_MAX_LEVEL
-   :swizzle-r    GL3/GL_TEXTURE_SWIZZLE_R
-   :swizzle-g    GL3/GL_TEXTURE_SWIZZLE_G
-   :swizzle-b    GL3/GL_TEXTURE_SWIZZLE_B
-   :swizzle-a    GL3/GL_TEXTURE_SWIZZLE_A
-   :wrap-s       GL2/GL_TEXTURE_WRAP_S
-   :wrap-t       GL2/GL_TEXTURE_WRAP_T
-   :wrap-r       GL2/GL_TEXTURE_WRAP_R})
+(defn texture-param->gl-texture-param
+  ^long [texture-param]
+  "This function translates Clojure keywords into OpenGL constants. You can use
+  the keywords in texture parameter maps."
```

Comment:
This string is supposed to be a docstring, but instead it is inside the function body. Move it above the arglist?

Reply:
Oops. Nice catch! I'll fix it!